app/forms/outgoings_form.rb

```diff
@@ -2,6 +2,9 @@ class OutgoingsForm
   include ActiveModel::Model
   include ActiveModel::Attributes
   include SessionPersistable
+  include NumberValidatable
```

Comment:
Also, I think we need to include this in `BenefitModel`, as that has a :gbp attribute as well.